akka-cluster/src/main/scala/akka/cluster/protobuf/ClusterMessageSerializer.scala

```diff
@@ -336,7 +336,15 @@ class ClusterMessageSerializer(val system: ExtendedActorSystem) extends BaseSeri
 
     def memberFromProto(member: cm.Member) =
       new Member(addressMapping(member.getAddressIndex), member.getUpNumber, memberStatusFromInt(member.getStatus.getNumber),
-        member.getRolesIndexesList.asScala.map(roleMapping(_))(breakOut))
+        rolesFromProto(member.getRolesIndexesList.asScala))
+
+    def rolesFromProto(roleIndexes: Seq[Integer], resultSoFar: Set[String] = Set.empty): Set[String] = {
+      val roles = roleIndexes.map(roleMapping(_)).toSet
+      roles.find(_.startsWith("team-")) match {
+        case Some(_) ⇒ roles
+        case None    ⇒ roles + "team-default"
+      }
```

Comment:
Agreed on using the constant, that wasn't on the `wip-multi-dc-dev` branch yet though I believe :).

Not sure if a loop will be much nicer here, after all we don't really expect many roles. Will give it a shot though.